pkg/scheduler/algorithmprovider/registry.go

```diff
@@ -1,178 +0,0 @@
-/*
```

Comment:
Keep this pkg for now, only remove the flag. The pkg will be removed in the same PR that adds the set of default plugins to the defaulting logic.

Reply:
/cc @yuzhiquan